README.md

```diff
@@ -529,6 +530,17 @@ console.log(JSON.stringify(work))
  */
 ```
 
+### Handle flow deletion
+
+Deleting a flow cannot be done by adding the flow in the `destructiveChanges.xml` and deploy.
+A [known issue](https://issues.salesforce.com/issue/a028c00000gAwixAAC/deletion-of-flow-metadata-through-destructive-changes-not-supported) exist to cover this feature.
+
+We know 2 ways to delete flow (documented [here](https://github.com/scolladon/sfdx-git-delta/issues/588#issuecomment-1534318729))
+- with `FlowDefinition` metadata, but is obsolete since v44 (and yet still working)
+- with flow status and purge of deactivated versions
```

Comment:
@scolladon I've just tested deploying the Flow with `Status=Draft` and `Status=Obsolete` and it didn't disable the Flow in my org.

This is also described here: https://github.com/scolladon/sfdx-git-delta/issues/588#issuecomment-1541167107

In my opinion the second way would be to **delete specific Flow Versions** (here v1 and v2) using a `destructiveChanges.xml` like:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
    <types>
        <members>Set_Account_Description-1</members>
        <members>Set_Account_Description-2</members>
        <name>Flow</name>
    </types>
    <version>61.0</version>
</Package>
```

Reply:
That's great combo, it could be the way to delete a flow with one go (until the v44 is decomissionated and FlowDefinition metadata is not available anymore).

The issue I see is it requires to know which flow versions to list in the destructiveChanges.xml... Which can be a complicated things to determine because it could not be the same in different environments. 

So I think it should not be handled by SGD.
It could be automatized by another plugin dedicated to deleting flow.

I'll update the documentation in the PR with your solution, thx